firmware/mw_dual_motor_torque_ctrl/src/dual_motor_torque_ctrl.c

```diff
@@ -341,6 +341,15 @@ QepIndexWatchdog_t gQepIndexWatchdog[2] = {
 		{.isInitialized = false, .indexError_counts = 0, .toggleBit = false}};
 
 
+// ! Time for slowly increasing the torque used for calibration.
```

Comment:
There should be no space between `//` and `!`

Reply:
Done.